ci/windows/build_cudax.ps1

```diff
@@ -3,8 +3,8 @@ Param(
     [Parameter(Mandatory = $false)]
     [Alias("std")]
     [ValidateNotNullOrEmpty()]
-    [ValidateSet(20)]
-    [int]$CXX_STANDARD = 20,
+    [ValidateSet(17, 20)]
```

Comment:
This was intentionally disabled, cudax chose to not support C++17 on MSVC when we first added it.

Reply:
Yes the main reason was that mdspan was not supported on MSVC C++17

That has been fixed now and we should work towards enabling MSVC C++17